src/main/java/Duke.java

```diff
@@ -1,10 +1,98 @@
+                printTaskList(jobCount, listTasks);
+//                System.out.println(Task.jobCount);
+            } else if (inputCommand.length() >= 5) {
+                /** Above codition used to prevent code for running for nonsensical commands or for commands where no parameters are defined*/
+                if (inputCommand.contains("done")) {
+                    int taskNumber = Integer.parseInt(inputCommand.substring(5, 6));
+                    listTasks[taskNumber - 1].markAsDone();
+                    System.out.println("Noiice! You're done with this Task. Good for you. I've marked that ure done with it.");
+                    System.out.println("\t[" + listTasks[taskNumber - 1].getStatusIcon() + "]\t" + listTasks[taskNumber - 1].description);
+                }
+                /** To add TODO to list */
+
+                else if ((inputCommand.contains("todo"))) {
+                    newTask(new Todo(splitInputs[1]));
+                    System.out.println("____________________________________________________________");
+                    System.out.println("\tTask added:\t" + listTasks[jobCount - 1].printDetails());
+                    System.out.println("\tNow you have " + jobCount + " tasks in the list.\n");
+                    System.out.println("____________________________________________________________");
+                }
+                /** To add Deadline to list */
+
+                else if ((inputCommand.contains("deadline"))) {
+                    String[] attributeFinder = splitInputs[1].split("/by");
+                    String deadlineName = attributeFinder[0];
+                    String deadlineBy = attributeFinder[1];
+                    newTask(new Deadline(deadlineName, deadlineBy));
+                    System.out.println("____________________________________________________________");
+                    System.out.println("\tTask added:\t" + listTasks[jobCount - 1].printDetails());
+                    System.out.println("\tNow you have " + jobCount + " tasks in the list.\n");
+                    System.out.println("____________________________________________________________");
+                }
+                /** To add event to list */
+                else if ((inputCommand.contains("event"))) {
+                    String[] attributeFinder = splitInputs[1].split("/at");
+                    String eventName = attributeFinder[0];
+                    String eventAt = attributeFinder[1];
+                    newTask(new Events(eventName, eventAt));
+                    System.out.println("____________________________________________________________");
+                    System.out.println("\tTask added:\t" + listTasks[jobCount - 1].printDetails());
+                    System.out.println("\tNow you have " + jobCount + " tasks in the list.\n");
+                    System.out.println("____________________________________________________________");
```

Comment:
Shouldn't it be better if you implement some SLAP and extract some of these out as methods?